### PR TITLE
Created Jumbotron component

### PR DESCRIPTION
**Pull request by wx02shi**
closing #4 
image

### CODE REVIEW

**Comment by functionstackx** [comment]
@williamsxz99  screenshot of component plz and more verbose and descriptive for yr PR description

**Comment by wx02shi** [comment]
mb forgot to add sc
image

**Comment by functionstackx** [comment]
> mb forgot to add sc
> image
> 
No big deal, but
Can you edit your PR description and add yr ss to the first thing instead of commenting 😁

File: components/Jumbotron.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import JumboStyles from '../styles/Jumbotron.module.css'
+import { Jumbotron, Container } from 'react-bootstrap'
+
+const Jumbo = () => {
+  return (
+    <Jumbotron fluid className="py-5" id={JumboStyles.imageHolder}>
+      <div>
```
Comment (LemonFace0309) [inline components/Jumbotron.js:8]: I don't think you really need this div

**Review by functionstackx — APPROVED** [review]
good job wiiliiam

File: components/Jumbotron.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import JumboStyles from '../styles/Jumbotron.module.css'
+import { Jumbotron, Container } from 'react-bootstrap'
```
Comment (LemonFace0309) [inline components/Jumbotron.js:3]: add ownership

**Pull request merged**